examples/seq2seq/test_seq2seq_examples.py

```diff
@@ -311,6 +312,58 @@ def test_run_eval(model):
         assert Path(output_file_name).exists()
         os.remove(Path(output_file_name))
 
+@slow
+@pytest.mark.parametrize("model", [pytest.param(T5_TINY), pytest.param(BART_TINY), pytest.param(MBART_TINY)])
```

Comment:
Can this just be one model?

Reply:
yes, let's merge this one and then I will rework it and run_eval to run one model non-slow and the rest slow.